editor/plugins/tiles/tiles_editor_plugin.cpp

```diff
@@ -108,7 +108,11 @@ void TilesEditorPlugin::_thread() {
 						Vector2i coords = tile_map->get_cell_atlas_coords(0, cell);
 						int alternative = tile_map->get_cell_alternative_tile(0, cell);
 
-						Vector2 center = world_pos - atlas_source->get_tile_data(coords, alternative)->get_texture_origin();
+						Vector2 center = world_pos;
+						if (atlas_source->has_tile(coords) && atlas_source->has_alternative_tile(coords, alternative)) {
+							center -= atlas_source->get_tile_data(coords, alternative)->get_texture_origin();
+						}
+
```

Comment:
The check should be expanded to the following two lines, otherwise they will spit errors in `get_tile_texture_region`